src/App.tsx

```diff
@@ -100,6 +97,7 @@ const App = (props: AppProps) => {
   );
 
   function updateCycles(newCycles: Cycle[]) {
```

Comment:
Looks like I didn't mention this earlier, but why we using `function` in the component?